Core/Src/subsystems/gimbal.cpp

```diff
@@ -59,11 +57,11 @@ void act() {
     }
 }
 
-double calculateAngleError(double currAngle, double targetAngle) {
-    /* Assumes currAngle and targetAngle are 0<=a<2pi */
```

Comment:
I think it makes sense to leave this in as someone quickly glancing at the function might not realize it.

Reply:
Resolved as per design change